tools/templates/channel_wrappers.tpl

```diff
@@ -0,0 +1,69 @@
+# This file auto-generated by `generate_schema_interface.py`.
+# Do not modify this file directly.
+
+import traitlets as T
+import pandas as pd
+
+from ...utils import parse_shorthand, infer_vegalite_type
+from ...utils import INV_TYPECODE_MAP, TYPE_ABBR
+
+from .._interface import Type
+
+{%- for object in objects %}{% for import in object.imports %}
+{%- set comma = joiner(", ") %}
+from {{ import.module }} import {% for name in import.names %}{{ comma() }}{{ name }}{% endfor %}
+{%- endfor %}{% endfor %}
+
+{% for object in objects %}
+class {{ object.name }}({{ object.base.name }}):
+    """Wrapper for Vega-Lite {{ object.base.name }} definition.
+    {{ object.base.description }}
+    Attributes
+    ----------
+    shorthand: Unicode
+        A shorthand description of the channel
+    {% for attr in object.base.attributes -%}
```

Comment:
OK, I think the blank lines are ok for now. If there are any other related
changes, why don't you submit an additional PR.

On Wed, Jun 8, 2016 at 9:04 PM, Jake Vanderplas <notifications@github.com>
wrote:

> In tools/templates/channel_wrappers.tpl
> <https://github.com/ellisonbg/altair/pull/116#discussion_r66379827>:
>
> > +from .._interface import Type
> > +
> > +{%- for object in objects %}{% for import in object.imports %}
> > +{%- set comma = joiner(", ") %}
> > +from {{ import.module }} import {% for name in import.names %}{{ comma() }}{{ name }}{% endfor %}
> > +{%- endfor %}{% endfor %}
> > +
> > +{% for object in objects %}
> > +class {{ object.name }}({{ object.base.name }}):
> > +    """Wrapper for Vega-Lite {{ object.base.name }} definition.
> > +    {{ object.base.description }}
> > +    Attributes
> > +    ----------
> > +    shorthand: Unicode
> > +        A shorthand description of the channel
> > +    {% for attr in object.base.attributes -%}
>
> I don't know how to do that.
>
> Actually, I guess it would just be an if-else statement in the jinja.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/ellisonbg/altair/pull/116/files/b7302c4a2ce079c2605e688ab168cd5c4668910e#r66379827>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABr0IqUV69Ve1M9FxK6rzRlzjGjFy6mks5qJ5DcgaJpZM4IorT7>
> .
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com